azext_connect/_params.py

```diff
@@ -33,6 +49,18 @@ def load_arguments(self, _):
         c.argument('password', options_list=['--password', '-pwd'], help='Password of the database. Only valid when auth type is secret')
         c.argument('keyvault', options_list=['--keyvault', '-kvt'], help='Keyvault name in the same resource group.')
 
+    with self.argument_context('cupertino webapp postgres') as c:
```

Comment:
Since the command contains three sub-groups, please sync with Xin and CLI team to make sure if this is a reasonable command to expose to the end-user for private preview

Reply:
@shizn You probably need to know this.